### ISSUE TITLE
[Typescript] TS2503: Cannot find namespace 'validator'

### ISSUE THREAD

**Issue by nmsbaka** [express-validator/express-validator#726]
Hi all,

Today, I upgrade express-validator to version 6.0.0 and get below notices. So I hope you help me to resolve this problem.

![image](https://user-images.githubusercontent.com/11207021/59910302-d63fd680-943b-11e9-9833-2ffde3d8cbbd.png)

```
[TypeScript] node_modules/express-validator/src/chain/sanitizers-impl.d.ts(15,30): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/sanitizers.d.ts(9,30): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(25,22): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(26,29): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(32,27): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(34,26): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(36,25): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(38,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(39,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(40,22): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(41,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(44,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(48,20): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(51,22): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(53,25): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(58,21): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(62,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(65,28): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(68,27): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(68,66): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(71,25): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(73,26): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(76,21): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators-impl.d.ts(77,22): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(17,22): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(18,29): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(24,27): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(26,26): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(28,25): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(30,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(31,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(32,22): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(33,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(36,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(40,20): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(43,22): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(45,25): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(50,21): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(54,23): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(57,28): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(60,27): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(60,66): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(63,25): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(65,26): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(68,21): error TS2503: Cannot find namespace 'validator'.
[TypeScript] node_modules/express-validator/src/chain/validators.d.ts(69,22): error TS2503: Cannot find namespace 'validator'.
```

**Comment by gustavohenke** [comment]
Thanks for the report! Seems like an oversight from me. Looking at a fix now!

**Comment by gustavohenke** [comment]
Fixed by a8ba124a8a86e7e9f9f579d766a7b8a4248409fc, and published as v6.0.1!
